javaagent-api/src/main/java/io/opentelemetry/javaagent/instrumentation/api/concurrent/ExecutorInstrumentationUtils.java

```diff
@@ -14,40 +14,68 @@
 /** Utils for concurrent instrumentations. */
 public class ExecutorInstrumentationUtils {
 
-  private static final ClassValue<Boolean> NOT_INSTRUMENTED_RUNNABLE_ENCLOSING_CLASS =
+  private static final ClassValue<Boolean> INSTRUMENTED_RUNNABLE_CLASS =
```

Comment:
Are changes in this class related to RxJava changes?

Reply:
Changes in this class caused the rxjava test to fail unexpectedly :( There's no way to verify the behavior of the rxjava instrumentation without the changes in this class (as we found in your muzzle PR) so they're somewhat tied.

Reply:
Though the change is larger than it could have been here since I figured I had too many `if` statements by now to not cache it